ehitchhiking-ui/src/app/app.component.html

```diff
@@ -1,2 +1,3 @@
-<!--<app-menu></app-menu>-->
+<app-menu></app-menu>
 <app-main-screen-component></app-main-screen-component>
+<app-blacklist></app-blacklist>
```

Comment:
Blacklist is a modal hence you should open it like a modal. Kindly look at docs https://material.angular.io/components/dialog/overview